doc/source/serve/tutorials/gradio.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
should we do myst instead?

Reply:
with myst the problem is that we don't get the Colab launch button (and binder is also strange in terms of integration). So I'd prefer ipynb, like in Tune.